content/automate/ManageIQ/System/Event/EmsEvent/Ansible_Tower.class/_missing.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
@lfu Talking to @Fryguy about this and it would be better to follow existing pattern which would mean that:

https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/embedded_ansible/automation_manager/event_parser.rb#L5
Should be changed to `EMBEDDEDANSIBLE`

https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/automation_manager/event_parser.rb#L5
Should be changed to `ANSIBLETOWER`

Then we would need to update this PR and https://github.com/ManageIQ/manageiq-content/pull/69 to match the new source changes.  (And use mixed-case in the automate class so it looks nicer like we do for `OpenStack`)